src/darwin/Framework/CHIP/MTROTAImageTransferHandler.mm

```diff
@@ -78,6 +89,11 @@ - (instancetype)initWithMTROTAImageTransferHandler:(MTROTAImageTransferHandler *
     VerifyOrReturnError(mDelegate != nil, CHIP_ERROR_INCORRECT_STATE);
     VerifyOrReturnError(mDelegateNotificationQueue != nil, CHIP_ERROR_INCORRECT_STATE);
 
+    mIsPeerNodeAKnownThreadDevice = [controller definitelyUsesThreadForDevice:mPeer.GetNodeId()];
+
+    uint16_t throttleInterval = chip::Platform::GetUserDefaultBDXThrottleIntervalForThreadInMSecs().value_or(kBdxThrottleDefaultIntervalInMsecs.count());
```

Comment:
Please remove the extra "chip::" prefix.  Also, GetUserDefaultBDXThrottleIntervalForThreadInMSecs should just be returning `optional<System::Clock::Milliseconds32>` and then you don't need this `count()` business or putting "InMsecs" in the name.

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/37562